definitions/json/CaseEvent/CaseEvent-Bundles.json

```diff
@@ -29,5 +29,20 @@
     "CallBackURLAboutToStartEvent": "",
     "CallBackURLAboutToSubmitEvent": "",
     "CallBackURLSubmittedEvent": ""
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "ID": "removeHearingBundles",
+    "Name": "Remove hearing bundles",
+    "Description": "Remove Hearing Bundles",
+    "DisplayOrder": 2,
```

Comment:
can you double check with Dan as to where this should be placed in the next steps menu? Don't think it should be 2nd

Reply:
updated: https://github.com/hmcts/et-ccd-definitions-englandwales/pull/879/commits/339db25881b020d26ab224618101b4cd53718c8d